src/BenchmarksApps/TechEmpower/Minimal/Minimal.csproj

```diff
@@ -1,15 +1,20 @@
-<Project Sdk="Microsoft.NET.Sdk.Web">
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
     <TargetFramework>net7.0</TargetFramework>
     <Nullable>enable</Nullable>
     <ImplicitUsings>enable</ImplicitUsings>
     <LangVersion>latest</LangVersion>
+    <UserSecretsId>38063504-d08c-495a-89c9-daaad2f60f31</UserSecretsId>
```

Comment:
```suggestion
```

Reply:
Why delete this? Having the project configured for user secrets makes it much easier to use.

Reply:
I'll read the docs to understand how it works. Since it worked "before" I don't see how it's better. Unless it's specific to local development.

Reply:
User secrets is a local development feature, yes. It means I can keep my local Postgres connection string the user secrets store for this project rather than having to edit settings files or configure environment variables.